querybook/webapp/components/DataDocSchedule/DataDocScheduleForm.tsx

```diff
@@ -284,6 +276,126 @@ export const DataDocScheduleForm: React.FunctionComponent<
     );
 };
 
+const NotifactionFormRow: React.FC<{
+    name: string;
+    index: number;
+    removeRow: (index: number) => void;
+    notificationRow: IDataDocKwargsNotifications;
+    notifiers: INotifier[];
+}> = ({ name, index, notificationRow, removeRow, notifiers }) => {
+    const { setFieldValue } = useFormikContext<IScheduleFormValues>();
+    const notificationFormName = `${name}[${index}]`;
+    const handleNotifyTo = useCallback((value) => {
+        setFieldValue(
+            `${notificationFormName}.config.to`,
+            value.split(',').map((v) => v.trim())
+        );
+    }, []);
+    const handleRemoveNotifier = useCallback((index) => removeRow(index), []);
+
+    const notifierOptions = useMemo(
+        () => notifiers.map((notifier) => notifier.name),
+        [notifiers]
+    );
+
+    const notifyOnOptions = useMemo(
+        () =>
+            getEnumEntries(NotifyOn).map(([key, value]) => ({
+                value,
+                label: key,
+            })),
+        []
+    );
```

Comment:
you can probably move this to the parent component and pass them as props, as all the notification rows will share the same options